themes/helm/layouts/partials/sidebar.html

```diff
@@ -21,20 +20,10 @@
           </a>
         </li>
 
-        {{ range $docsSections }}
-        {{ range .Sections }}
-        {{ .Render "section-link" }}
-
-        {{ range .Sections }}
-        {{ .Render "section-link" }}
-        {{ end }}
-        {{ end }}
-
         {{ range .Pages }}
```

Comment:
Oh, I think I see... use `.Site.Pages`. Missed this before.

Reply:
site.Pages also won’t work because that will include blog posts. I’ll submit a PR to fix this, as sections are tricky.

Reply:
@mattfarina @lucperkins I've updated with the desired result as follows:

```
{{ range (where .Site.Pages "Section" "docs") }}
          {{ range .Sections }}
```

Preview here: https://deploy-preview-528--helm-merge.netlify.com/docs/
There's probably a more graceful way to do this though, so any input is helpful.

Reply:
Oops, I also addressed this in #540. Perhaps we can merge that and you can merge your alias changes from this PR?

Reply:
Oops, I submitted a similar fix simultaneously. I'll close that PR (#540).